nowcasting_dataset/config/model.py

```diff
@@ -364,7 +388,8 @@ class GSP(DataSourceMixin, StartEndDatetimeMixin):
         description="The number of GSP samples per example. "
         "If there are less in the ROI then the data is padded with zeros. ",
     )
-    gsp_image_size_pixels: int = IMAGE_SIZE_PIXELS_FIELD
+    gsp_image_size_pixels_height: int = IMAGE_SIZE_PIXELS_FIELD
```

Comment:
suprised GSP uses image size?

Reply:
Yeah, quite a few inherit from ImageDataSource, which is where I ended up changing the code.